config.json

```diff
@@ -81,7 +81,7 @@
     "show_accept_googlepay": false,
     "show_accept_klarna": false,
     "show_product_details_tabs": true,
-    "show_product_reviews_tabs": false,
+    "show_product_reviews": true,
```

Comment:
I have a couple of questions here;
- Is there a reason to change the key name here specially when for product desc we still use `show_product_details_tabs` ? I like the old name was explicit and clear.
- Why are we changing the default from `false` to `true` ? Is this a product decision cc @bc-as  ?

Reply:
* in theme builder we have checkbox with name "Show product reviews" but previous logic was depending on show_products_reviews_tab. Now logic have been changed and "Show product reviews" checkbox is responsible for product reviews display. So, for better code semantic I decided to change its name to current
* In case of false we won't see any product reviews, so in case of testing we need to set it to true anyway. It's not a decision of @bc-as

Reply:
@junedkazi @BC-tymurbiedukhin 

- We should keep the same key here for consistency and to ensure that merchants don't have an issue when they update their theme.  That being said, the option in page builder/theme editor is "Show product reviews" and not "Show product reviews tab".
- Product reviews should be turned on by default unless the merchant wants to opt out